Annex60/Fluid/BaseClasses/PartialThreeWayResistance.mo

```diff
@@ -78,87 +76,44 @@ partial model PartialThreeWayResistance
             0}},                                                                      color={0,127,255}));
   else
-    if not dynamicBalance then
-       connect(res1.port_b, port_internal) annotation (Line(
-      points={{-40,0},{-40,60},{0,60}},
-      color={0,127,255}));
-    else
-       connect(res1.port_b, vol.ports[1]) annotation (Line(
+    connect(res1.port_b, vol.ports[1]) annotation (Line(
       points={{-40,0},{-2.66667,0}},
       color={0,127,255}));
-    end if;
     connect(port_1, res1.port_a) annotation (Line(points={{-100,0},{-100,0},{-60,0}}, color={0,127,255}));
   end if;
 
   if portFlowDirection_2==Modelica.Fluid.Types.PortFlowDirection.Leaving then
-    if not dynamicBalance then
-       connect(res2.port_a, port_internal) annotation (Line(
-      points={{60,0},{60,60},{0,60}},
-      color={0,127,255}));
-    else
-       connect(res2.port_a, vol.ports[2]) annotation (Line(
+    connect(res2.port_a, vol.ports[2]) annotation (Line(
       points={{60,0},{2.22045e-16,0}},
       color={0,127,255}));
-    end if;
     connect(port_2, res2.port_b) annotation (Line(points={{100,0},{100,0},{40,0}},    color={0,127,255}));
   else
-    if not dynamicBalance then
-       connect(res2.port_b, port_internal) annotation (Line(
-      points={{40,0},{40,60},{0,60}},
-      color={0,127,255}));
-    else
-       connect(res2.port_b, vol.ports[2]) annotation (Line(
+    connect(res2.port_b, vol.ports[2]) annotation (Line(
       points={{40,0},{2.22045e-16,0}},
       color={0,127,255}));
-    end if;
     connect(port_2, res2.port_a) annotation (Line(points={{100,0},{100,0},{60,0}},    color={0,127,255}));
   end if;
 
   if portFlowDirection_3==Modelica.Fluid.Types.PortFlowDirection.Leaving then
-    if not dynamicBalance then
-       connect(res3.port_a, port_internal) annotation (Line(
-      points={{-4.44089e-16,-60},{20,-60},{20,60},{0,60}},
+    connect(res3.port_a, vol.ports[3]) annotation (Line(
+      points={{-6.66134e-016,-60},{0,-60},{0,0},{2.66667,0}},
       color={0,127,255}));
-    else
-       connect(res3.port_a, vol.ports[3]) annotation (Line(
-      points={{-6.66134e-16,-60},{0,-60},{0,0},{2.66667,0}},
-      color={0,127,255}));
-    end if;
     connect(port_3, res3.port_b) annotation (Line(points={{0,-100},{0,-100},{0,-40}}, color={0,127,255}));
   else
-    if not dynamicBalance then
-       connect(res3.port_b, port_internal) annotation (Line(
-      points={{4.44089e-16,-40},{20,-40},{20,60},{0,60}},
-      color={0,127,255}));
-    else
-       connect(res3.port_b, vol.ports[3]) annotation (Line(
-      points={{4.44089e-16,-40},{0,-40},{0,0},{2.66667,0}},
+    connect(res3.port_b, vol.ports[3]) annotation (Line(
+      points={{4.44089e-016,-40},{0,-40},{0,0},{2.66667,0}},
       color={0,127,255}));
-    end if;
     connect(port_3, res3.port_a) annotation (Line(points={{0,-100},{0,-100},{0,-60}}, color={0,127,255}));
   end if;
    annotation (
```

Comment:
@rubenbaetens : Please also add revision notes so we can easily track what revisions were made to the file.